repository/provider_queries.py

```diff
@@ -3,8 +3,12 @@
 from models.provider import Provider
 
 
-def find_provider_by_id(provider_id: int) -> Provider:
-    return Provider.query.filter_by(id=provider_id).first()
+def find_enabled_provider_for_pro_by_id(provider_id: int) -> Provider:
```

Comment:
afin d'homogénéiser :
```suggestion
def get_provider_enabled_for_pro_by_id(provider_id: int) -> Provider:
```

Reply:
Yes !